ethpm/package.py

```diff
@@ -142,6 +142,26 @@ def name(self) -> str:
     def version(self) -> str:
         return self.package_data["version"]
 
+    #
+    # Build Dependencies
+    #
+
+    def get_build_dependencies(self) -> "Dependencies":
+        """
+        Retrieve instance of `Dependencies` (i.e. build dependencies)
+        belonging to this Package.
+        """
+        validate_build_dependencies_are_present(self.package_data)
+
+        dependencies = self.package_data["build_dependencies"]
+        validate_build_dependencies(dependencies)
+        dependency_packages = {
+            dependency: Package.from_ipfs(uri)
```

Comment:
I think we'll want to be a bit more fancy here and catch any failures in this `Package.from_ipfs(uri)` in such a way that we can raise an error with more useful information.

Something like:

> Failed to retrieve build dependency `safe-math-lib` from URI `ipfs://Qm....`: Got error: <the-error>